utils/warehouseHelpers.js

```diff
@@ -0,0 +1,54 @@
+// Function to validate the request body for a new warehouse item
+async function validateNewWarehouseItem({
+  warehouse_name,
+  address,
+  city,
+  country,
+  contact_name,
+  contact_position,
+  contact_phone,
+  contact_email,
+}) {
+  if (
+    !warehouse_name ||
+    !address ||
+    !city ||
+    !country ||
+    !contact_name ||
+    !contact_position ||
+    !contact_phone ||
+    !contact_email
+  ) {
+    return { isValid: false, message: "Missing required fields" };
+  }
+  // Use regex to validate that phone number is a 10 digit number
+  if (!/^\d{10}$/.test(contact_phone)) {
+    return {
+      isValid: false,
+      message: "Please enter a valid 10-digit phone number",
+    };
+  }
+
+  // Use regex to validate that email has correct format
```

Comment:
we don't need to validate email format, let the front end handle that, it would be easier to use input type email on the FE

Reply:
Its always okay to validate email or any validations in BE too since BE is the last line of defense in case FE is not implemented by the same person.